cmake/sysbuild/sdp.cmake

```diff
@@ -2,48 +2,20 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-# Update eGPIO images KConfig based on SDP configuration.
-#
-# Usage:
-#   egpio_update_kconfig()
-#
-function(egpio_update_kconfig)
+function(sdp_apply_overlays overlay)
```

Comment:
Is this overlay argument needed? I do not see it used anywhere.
Also, I can see that this file is modified again two commits later and there is an argument, but it is named differently. Maybe those commits should be merged? Treewide and cmake, to be specific

Reply:
Fixed, but left the commits separate - they touch upon different issues.